src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesTask.groovy

```diff
@@ -53,7 +54,8 @@ class DependencyUpdatesTask extends DefaultTask {
   boolean checkForGradleUpdate = true
 
   Object outputFormatter = 'plain'
-  Action<? super ResolutionStrategyWithCurrent> resolutionStrategyAction = null
+  Closure resolutionStrategy = null;
```

Comment:
If it was unintentional, then I would not worry about supporting this till the end of times. I get that you don't want to suddenly break the build for some people. In the future, it could be marked as deprecated with explanation + warning in the readme.

Otherwise LGTM

Reply:
This was hidden in the guts of Gradle, but now shows up as a deprecated warning and the message visible when using `--warning-mode=all`.